### ISSUE TITLE
Training Code

### ISSUE THREAD

**Issue by SanthoshSandi** [devopsbay/face-mask-detection#1]
Can we get the training scripts?
